inscriptions/ISic030234.xml

```diff
@@ -0,0 +1,246 @@
+                <language ident="he">Hebrew</language> 
+                <language ident="phn">Phoenician</language>
+                <language ident="xpu">Punic</language>
+                <language ident="osc">Oscan</language> 
+                <language ident="xly">Elymian</language> 
+                <language ident="scx">Sikel</language>  
+                <language ident="sxc">Sikan</language>  
+            </langUsage>
+            <textClass>
+                <keywords scheme="http://www.eagle-network.eu/voc/typeins.html">
+                    <term ana="#function.tessera.civic" ref="https://www.eagle-network.eu/voc/typeins/lod/130.html">tessera (civic)</term>
+                </keywords>
+            </textClass>
+        </profileDesc>
+        <revisionDesc status="draft">
+            <listChange>
+                <change when="2020-10-05" who="#SS">Simona Stoyanova normalised Unicode</change>
+                <change when="2020-10-08" who="#SS">Simona Stoyanova updated list of languages</change>
+                <change when="2021-12-10" who="#VM">Valentina Mignosa added metadata and text</change>
+            </listChange>
+        </revisionDesc>
+    </teiHeader>
+    <facsimile>
+        <surface/> 
+        <!--delete empty surface above and uncomment template below if needed-->
+        <!--
+  <surface type="front">
+    <graphic n="screen" url="{concat('ISic', $num, '_tiled.tif')}"
+      height="4912px" width="7360px">
+      <desc>Add Description Here</desc>
+    </graphic>
+    <graphic n="print" url="{concat('ISic', $num, '.jpg')}" height="3680px" width="5520px">
+      <desc>Add Description Here</desc>
+    </graphic>
+  </surface>
+
+    -->
+    </facsimile>
+    <text>
+        <body>
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM">
+                <div type="textpart" subtype="face" n="a">
+                    <ab>
+                        <lb n="1"/><handShift new="#h1"/><orig>Α</orig><gap reason="lost" extent="unknown" unit="character"/>
+                        <lb n="2"/><persName><name type="patronym" cert="low"><seg part="I">Αγεσ</seg><gap reason="lost" extent="unknown" unit="character"/></name></persName> <gap reason="lost" extent="unknown" unit="character"/>
+                        <lb n="3"/><handShift new="#h2"/><gap reason="lost" extent="unknown" unit="character"/><orig>Λ</orig><gap reason="lost" extent="unknown" unit="character"/>
+                    </ab>
+                </div>
+                <div type="textpart" subtype="face" n="b">
+                    <ab>
+                        <lb n="1"/><handShift new="#h1"/><num value="6">hέκτ<supplied reason="lost">α</supplied></num>
+                        <lb n="2" <!-- maybe sth as: style="text-direction:reverse"?-->/><handShift new="#h2"/><gap reason="lost" extent="unknown" unit="character"/><orig>Σ</orig> <!-- this line can be read turning the tessera upside down. How can we flag this? -->
```

Comment:
is it enough to do  `<lb n="2" style="text-direction:r-to-l"/>` ? I appreciate this doesn't capture that it's also the other way up, which formally one captures with `<hi rend="inverted">` instead / in addition.